tree/treeplayer/src/TBranchProxy.cxx

```diff
@@ -256,7 +256,11 @@ Bool_t ROOT::Detail::TBranchProxy::Setup()
       // This does not allow (yet) to precede the branch name with
       // its mother's name
       fBranch = fDirector->GetTree()->GetBranch(fBranchName.Data());
-      if (!fBranch) return false;
+      if (!fBranch) {
+         auto treeName = fDirector->GetTree()->GetName();
+         ::Error("TBranchProxy::Setup", "%s", Form("Unable to find branch %s in tree %s.\n",fBranchName.Data(), treeName));
```

Comment:
Note that the Error routine adds a newline (so this prints 2 newlines :) ).

Reply:
corrected, thanks.
What about the failure?

Reply:
I discovered that 60% of them are a bug in MakeProxy and 40% are 'expected' (asking for a branch that does not exist because it is 'ellided').